test/fixtures/es-modules/esm-top-level-await.mjs

```diff
@@ -0,0 +1,9 @@
+import { setTimeout } from 'node:timers/promises';
+
+await new Promise(async (resolve) => {
```

Comment:
Why isn't this just
```js
await setTimeout(5);
console.log(1);
console.log(2);
```

?

Reply:
(like, if it's just a fixture for TLA why the extra `new Promise`?)

Reply:
I’d guess it’s because the `await new Promise` was an example that I offered, and my code didn’t use `import { setTimeout } from 'node:timers/promises'`; but the file where my example was put had that `import`. So I was assuming a `setTimeout` that didn’t return a promise.